test/inductor/test_multi_kernel.py

```diff
@@ -0,0 +1,104 @@
+# Owner(s): ["module: inductor"]
+
+import torch
+from torch import nn
+from torch._dynamo.testing import reset_rng_state
+
+from torch._inductor import config
+from torch.nn import functional as F
+from torch.testing._internal.common_utils import TestCase
+from torch.testing._internal.inductor_utils import HAS_CUDA
+
+config.triton.multi_kernel = 1
+config.benchmark_kernel = True
+config.compile_threads = 1
+
+
+class TransformerSnippet(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.ln1 = nn.LayerNorm(64)
+        self.ln2 = nn.LayerNorm(64)
+
+    def forward(self, x1, x2):
+        x1 = F.dropout(x1, 0.1)
+        x2 = F.dropout(self.ln1(x2), 0.1)
+
+        return self.ln2(x1 + x2)
+
+    def example_inputs(self):
+        return (torch.randn(2, 64).cuda(), torch.randn(2, 64).cuda())
+
+
+class MultiKernelTest(TestCase):
```

Comment:
These tests aren't checking that multikernel support actually worked.
1) add a test that asserts something about the output code containing two kernels
2) perhaps add a way to force once kenrel to get picked and assert it was picked